pkgs/applications/editors/jupyter/console.nix

```diff
@@ -0,0 +1,43 @@
+# Jupyter console with the given kernel definitions
+
+{ python3
+, jupyter-kernel
+, lib
+}:
+
+let
+  mkConsole = {
+    definitions ? jupyter-kernel.default
+    , kernel ? null
+  }:
+    let
+      jupyterPath = (jupyter-kernel.create { inherit definitions; });
+    in
+      (python3.buildEnv.override {
+        extraLibs = [python3.pkgs.jupyter_console];
+        makeWrapperArgs = [
+          "--set" "JUPYTER_PATH" "${jupyterPath}"
+        ] ++ lib.optionals (kernel != null) [
+          "--add-flags" "--kernel"
+          "--add-flags" kernel
+        ];
+      }).overrideAttrs (oldAttrs: {
+        # To facilitate running nix run .#jupyter-console
```

Comment:
The correct way to fix this, is to set meta.mainProgram

Reply:
Done